hw/ip/flash_ctrl/dv/cov/flash_ctrl_cov.vRefine

```diff
@@ -36,6 +36,8 @@
     <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.a_ready&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
     <rule ccType="inst" ccfFlagsMask="4205506-42" domain="icc" entityName="flash_ctrl/u_flash_ctrl_erase/1" entityType="top-expr" excTime="1653576309" file="1" im-checksum="95618010" line="42" name="exclude" reviewer="unknown" text="op_start_i &amp; op_addr_oob_i" ung="0" user="0" vscope="default"></rule>
     <rule ccType="inst" ccfFlagsMask="4205506-42" domain="icc" entityName="flash_ctrl/u_flash_ctrl_erase/7" entityType="top-expr" excTime="1653576316" file="1" im-checksum="95618010" line="56" name="exclude" reviewer="unknown" text="op_start_i &amp; (~ op_addr_oob_i)" ung="0" user="0" vscope="default"></rule>
+    <rule ccType="inst" ccfFlagsMask="4205506-42" domain="icc" entityName="flash_ctrl/u_flash_ctrl_erase/2" entityType="top-expr" excTime="1653908436" file="1" im-checksum="95618010" line="45" name="exclude" reviewer="unknown" text="flash_req_o &amp; (flash_done_i | oob_err)" ung="0" user="2" vscope="default"></rule>
```

Comment:
hmmm... is there any way to do this is a "line agnostic" way? As it stands right now, it seems like anytime there are updates this exclusion may no longer apply? Is that right?

Reply:
@cindychip : Do you know the answer of Tim's question since I took your vRefine file as an example? RTL is D2S currently.

Reply:
D2S doesn't mean there are no bug fixes, and that could very easily move lines around.

Reply:
Good point, I do not know that on top of my head. We are not very familiar with Xcelium. We could reach out to AE or wonder if @rasmus-madsen knows.